packages/icons/icons.json

```diff
@@ -4737,5 +4737,19 @@
         "tags": "page break",
         "group": "editor",
         "codepoint": 62373
+    },
+    {
+        "displayName": "Branch protected",
+        "iconName": "branch-protected",
```

Comment:
Can we name these `branch-locked` and `branch-unlocked`? Naming the objects (e.g. "lock") vs. semantics/concepts (e.g. "protection") lessens the chance that we'll see a naming collision here in the future. For example, the concept of "protecting" something could also be represented with a "shield".